transformedTests/tsconfig.json

```diff
@@ -0,0 +1,14 @@
+{
+    "compilerOptions": {
+        "module": "commonjs",
+        "target": "ESNext",
+        "sourceMap": true,
+        "declarationMap": false,
+        "declaration": false,
+        "plugins": [
+            {
+                "transform": "typescript-rtti/dist/transformer"
```

Comment:
set this to `"../dist/transformer"`

Reply:
Works, as is, after the change in package.json